cmd/prysmctl/withdrawal/withdrawal.go

```diff
@@ -0,0 +1,165 @@
+package withdrawal
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/fs"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/logrusorgru/aurora"
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/v3/beacon-chain/rpc/apimiddleware"
+	"github.com/prysmaticlabs/prysm/v3/io/prompt"
+	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+	"go.opencensus.io/trace"
+)
+
+const (
+	basePath = "/eth/v1"
+	apiPath  = "/beacon/pool/bls_to_execution_changes"
+)
+
+func setWithdrawalAddress(c *cli.Context, r io.Reader) error {
```

Comment:
Update to plural if you agree with my previous comment.

Reply:
will update